test/integration/util/util.go

```diff
@@ -102,15 +102,8 @@ func (m *MinishiftRunner) IsCDK() bool {
 	return strings.Contains(cmdOut, "minishift setup-cdk [flags]")
 }
 
-func (m *MinishiftRunner) EnsureRunning() {
-	if m.GetStatus() != "Running" {
-		m.Start()
-	}
-	m.CheckStatus("Running")
-}
-
 func (m *MinishiftRunner) IsRunning() bool {
-	return m.GetStatus() == "Running"
+	return strings.Contains(m.GetStatus(), "Running")
```

Comment:
In this case the status can be checked as `contains/matches "Minishift: Running" AND contains/matches "OpenShift: Running"`, but I think our test is more about Minishft...

So maybe we should have two different functions for this... WDYT @agajdosi ?
  * `IsOpenShiftRunning`
  * `IsMinishiftRunning`